# BMSAlarmWarning.msg - BMS报警和警告信息
# BMS报警警告信息
uint8 alarm1            # 报警字节1
uint8 alarm2            # 报警字节2
uint8 warn1             # 警告字节1
uint8 warn2             # 警告字节2
bool over_voltage_alarm    # 过压报警
bool under_voltage_alarm   # 欠压报警
bool high_temp_alarm       # 高温报警
bool low_temp_alarm        # 低温报警
bool discharge_overcurrent_alarm  # 放电过流报警
bool charge_overcurrent_alarm     # 充电过流报警
bool over_voltage_warning         # 过压警告
bool under_voltage_warning        # 欠压警告
bool high_temp_warning            # 高温警告
bool low_temp_warning             # 低温警告
bool discharge_overcurrent_warning # 放电过流警告
bool charge_overcurrent_warning    # 充电过流警告
bool has_alarm                     # 是否有报警
bool has_warning                   # 是否有警告